### PR TITLE
[Added]: Showing html pages

### PR DESCRIPTION
**Pull request by agnivade**
- Querying all mDNS services and getting their data.
- Now showing urls for all origins and all cache servers.
- And templatizing the HTML to point to the correct cache server.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,17 +1,11 @@
 package main
```
Comment (pdey) [inline main.go:14]: Add overview comments in this file

File: registry.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+	mux.Handle("/static/", fs)
+	mux.HandleFunc("/stream", r.streamHandler)
+	mux.HandleFunc("/", r.homeHandler)
+
+	return r
+}
+
+// Start the registry.
+func (r *registry) Start() {
```
Comment (pdey) [inline registry.go:65]: Does this periodically read mDNS stream?

**Review by pdey — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (agnivade) [reply main.go:14]: done

File: registry.go (hunk 2 above)
Comment (agnivade) [reply registry.go:65]: No. Will do in the next commit.

**Review by pdey — APPROVED** [review]

**Pull request merged**